boa_engine/src/builtins/array/array_iterator.rs

```diff
@@ -17,8 +17,11 @@ use boa_profiler::Profiler;
 #[derive(Debug, Clone, Finalize, Trace)]
 pub struct ArrayIterator {
     array: JsObject,
+    #[unsafe_ignore_trace]
     next_index: u64,
+    #[unsafe_ignore_trace]
     kind: PropertyNameKind,
+    #[unsafe_ignore_trace]
```

Comment:
Do we need the `#[unsafe_ignore_trace]` on `u64` and `bool`? Those should already be `unsafe_empty_trace` right?

Reply:
Yes, I will remove this, it will probably get optimized out by the compiler anyways.